packages/application-shell/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -1,55 +1,23 @@
-import { useState, lazy } from 'react';
-import { useFeatureToggle } from '@flopflip/react-broadcast';
+import { useEffect } from 'react';
 import { themesOverrides } from '@commercetools-frontend/application-components';
-import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
 import { ThemeProvider } from '@commercetools-uikit/design-system';
 import { STORAGE_KEYS } from '../../constants';
-import { UI_REDESIGN } from '../../feature-toggles';
-
-const NewDesignReleaseInfoDialog = lazy(
-  () => import('./new-design-release-info-dialog')
-);
-
-type TAdditionalEnvironmentValues = { disableRedesignAnnouncement: boolean };
-type TSelectedEnvironmentValues = boolean | undefined;
-function useDisableRedesignAnnouncement() {
-  const disableRedesignAnnouncement = useApplicationContext<
-    TSelectedEnvironmentValues,
-    TAdditionalEnvironmentValues
-  >((context) => context.environment.disableRedesignAnnouncement);
-
-  return Boolean(disableRedesignAnnouncement);
-}
 
 const ThemeSwitcher = () => {
-  const isNewThemeEnabled = useFeatureToggle(UI_REDESIGN);
-  const isRedesignAnnouncementDisabled = useDisableRedesignAnnouncement();
-  const theme = isNewThemeEnabled ? 'test' : 'default';
-  const [
-    hasUserSeenNewDesignReleaseNotificationDialog,
-    setHasUserSeenNewDesignReleaseNotificationDialog,
-  ] = useState(
-    window.localStorage.getItem(
-      STORAGE_KEYS.IS_NEW_DESIGN_RELEASE_NOTIFICATION_CLOSED
-    ) === 'true'
-  );
+  // This helps decoupling app-kit update from ui-kit update
+  // TODO: Remove after ui-kit redesign cleanup has been done
+  const theme = 'test';
```

Comment:
If someone updates `app-kit` before updating `ui-kit`, we will be using the new theme styles by default for `app-kit` but we need to make sure we still tell `ui-kit` we want the new theme.
Once they update `ui-kit`, passing it `test` as the theme will just show a warning in the console but it will be ignored and the `default` one will be used.

We can also wait for the new version of `ui-kit` to be released and update `app-kit` to use it as the minimum dependant versions.